react/src/components/Totals.tsx

```diff
@@ -33,6 +31,17 @@ function Totals (props: TotalsProps) {
     overpaymentEmployer
   } = useClassOneTotals()
 
+  const context: Context<any> =
```

Comment:
Should we not import the Directors and Class1 interfaces and use these as the required types?

Reply:
yes - this is a good point but we can't use dynamic types here like Context<ClassOneContext | DirectorsContext> because the native useContext hook will not accept this type - I think <any> is a temporary fix for now and arises because two calcs are sharing the same totals component - we do need to think about a better design and might be good to see if we need to use this component for any other calculators also...

Reply:
Thanks Ali